WordPress/Classes/ViewRelated/Site Creation/WebAddress/WebAddressWizardContent.swift

```diff
@@ -262,7 +262,13 @@ final class WebAddressWizardContent: UIViewController {
         }
 
         let textField = header.textField
-        guard let inputText = textField.text, !inputText.isEmpty else {
+
+        let inputText: String
+        if let text = textField.text, !text.isEmpty {
+            inputText = text
+        } else if let text = siteCreator.information?.title, !text.isEmpty {
+            inputText = text
```

Comment:
I think this is on the right track, but unfortunately in my testing, I'm not seeing the text field display the title from the site information screen.

Reply:
The text field doesn't appear to become first responder (i.e., gain "focus") here either.

Reply:
@stevebaranski I'm sorry I should've added screenshots to portray what the intended behavior was.

The title from basic information should not be reflected in the text field. We simply use the title to fetch domain suggestions in the background. Also we don't want the textfield as the first responder because it hides some of the suggestions (as shown below):

<img width="541" alt="screen shot 2019-02-08 at 10 14 12 am" src="https://user-images.githubusercontent.com/1158819/52497248-716d9900-2b8a-11e9-85a8-4b2c41e9716a.png">

Reply:
@kwonye I wasn't aware of a requirement like this. Moreover, I'm concerned about this proposed user experience.

If the search is implicit, and the placeholder text is still visible, and a user wants to further refine this implicit search, how do they do so? Do they tap _Back_? If they modify the search, how do they get back to this default result set?

In any case, we should probably reach consensus on this before evaluating the PR further.

Reply:
> If the search is implicit, and the placeholder text is still visible, and a user wants to further refine this implicit search, how do they do so? Do they tap _Back_?

Oh interesting, I never thought about it this way. For me it was fairly obvious to tap the text field and start typing for anything more relevant. Yours is a good perspective though.

> If they modify the search, how do they get back to this default result set?

Good point...

@SylvesterWilmott any thoughts? and cc @malinajirka in case there are any changes made that should be reflected on Android.

Reply:
>If the search is implicit, and the placeholder text is still visible, and a user wants to further refine this implicit search, how do they do so?

>If they modify the search, how do they get back to this default result set?

For context, this is quite close to the current behaviour of the domain search screen.

- Implicit suggestions are based on site title
- If no suggestions are returned then show an empty state
- If I type in the field a search can be performed
- If I clear the field and search again I am returned to the suggested domains

The only differences in our case is that if no results are returned we don't show an empty state and we update the results as you type in the field.

Reply:
> For me it was fairly obvious to tap the text field and start typing for anything more relevant.

@kwonye is this sufficiently obvious when you're using a screen reader?

Reply:
> For context, this is quite close to the current behaviour of the domain search screen...The only differences in our case is that if no results are returned we don't show an empty state and we update the results as you type in the field.

@SylvesterWilmott In the prior experience, the prompt text is small and remains onscreen while the user interacts with the feature. In the new experience, when a user interacts with the text field, we conceal the header and shift up the content to make room for additional results.

In reviewing this particular PR, I observed that the change proposed by this PR did not work consistently with this new precedent.

Reply:
> @kwonye is this sufficiently obvious when you're using a screen reader?

That's a good point. @SylvesterWilmott I do think @stevebaranski brings up a valid case and I will say it makes the code much simpler if we just have the text always reflect what's being searched. And because the 'x' is there to cancel out the text, it should be fairly easy for the user to start typing away a new request.